request/xhr.js

```diff
@@ -34,7 +34,9 @@ define([
 	has.add('native-xhr2-blob', function(){
 		if(!has('native-response-type')){ return; }
 		var x = new XMLHttpRequest();
-		x.open('GET', '/', true);
+		// URL should be fully-qualified as XMLHTTPRequest within a Worker
+		// created from a Blob does not support relative URL paths
+		x.open('GET', 'https://dojotoolkit.org/', true);
```

Comment:
Correct - the URL used is _almost_ entirely irrelevant since no request is actually sent, but it turns out it is relevant in some cases, such as when a relative URL cannot be resolved and will throw an error immediately upon calling `XMLHttpRequest#open`.

We don't need a reachable URL, we just need a parsable URL.